roles/install/handlers/main.yml

```
@@ -0,0 +1,19 @@
+- name: activate_pref_tuning
+  command: satellite-installer
+
+- name: firewalld_reload
+  systemd:
+    name: firewalld
+    state: reloaded
+
+- name: run_installer
+  command: >-
+    satellite-installer --scenario satellite
```

1. Is there any reason why the satellite installer below is hardcoded to `satellite` (what about capsule or other scenarios)?
2. Why are we using this and not deferring to a source `foreman_installer` task or dev source installer?
* https://github.com/theforeman/forklift/blob/master/roles/foreman_installer/tasks/install.yml

```suggestion
    satellite-installer --scenario {{ satellite_scenario }}
```

---

Have a capsule role that will be uploaded

---

changed the var to a variable